app/client/components/cancel/gw/gwCancellationFlowStart.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { trackEvent } from "../../analytics";
+import { PageContainerSection } from "../../page";
+import { hrefStyle } from "../cancellationConstants";
+
+const trackCancellationClickEvent = (eventLabel: string) => () =>
+  trackEvent({
+    eventCategory: "cancellation",
+    eventAction: "click",
+    eventLabel
+  });
+
+export const gwCancellationFlowStart = () => (
+  <PageContainerSection>
+    <h3>
+      We’re sorry to hear you’re thinking of cancelling your Guardian Weekly
+      subscription.
+    </h3>
+
+    <p>
+      Your support means The Guardian can remain editorially independent, free
+      from the influence of billionaire owners and politicians. This enables us
+      to give a voice to the voiceless, challenge the powerful and hold them to
+      account. Our readers’ support helps us to keep our journalism free of a
```

Comment:
Hi :)

Is the pluralisation of readers correct here?

Reply:
@JessL123 whilst I think this is correct, I don't like this sentence, would personally prefer something like...
```suggestion
      account. The support from our readers helps us to keep our journalism free of a
```

Reply:
Yup reckon your suggestion reads better - importance should be on "support" not "readers" so putting it at the start of the sentence works better.

Reply:
done - https://github.com/guardian/manage-frontend/pull/391/commits/b97e5f9363a794fce2d75571ca70c30758c86316